airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1595,17 +1595,17 @@ components:
     ConnectionCreate:
       type: object
       required:
-        - defaultNamespace
+        - prefix
```

Comment:
shouldn't this line be deleted? putting it in the required array makes it required. we want it to be optional. same of all other structs in this file.
```suggestion
```